apps/src/block_utils.js

```diff
@@ -747,6 +749,10 @@ exports.createJsWrapperBlockCreator = function (
       });
     }
     const inputConfigs = determineInputs(blockText, inputs, strictTypes);
+    const inputRows = groupInputsByRow(inputConfigs, inputTypes);
+    if (inputRows.length === 1) {
```

Comment:
Hmm, are you sure this is the right metric?  A lot of Blockly built-in blocks have multiple inputs that are all external:

![image](https://user-images.githubusercontent.com/413693/40145877-5b7eb6be-5918-11e8-8f54-d3767235805c.png)

It's almost like the "80-character max width" of Blockly programs 😛.  Sometimes the blocks are uglier, but it reduces the chance of horizontal overflow.

Reply:
This is just the case in which I **force** you to turn off `inline` on a block. You're always free to add `inline: false` to the block definition to get blocks like that, but I think that should still be optional.